ui/CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+# Contributing to the UI
+
+This part of the project contains the user interface for the Kafka console, developed using Next.js and the [PatternFly](https://patternfly.org) UI library.
+
+## Prerequisites
+
+Please make sure you have working installations of:
+
+- node (v18+)
+- npm (v10+)
+
+To run a development version of the UI working in all its sections, you will need to install the console on a development cluster first. Please refer to the [install/README.md](../install/README.md) file for detailed instructions about how to do it.
```

Comment:
I'm confused, the `./install/README` file mentions yet another way of installing things, and is not clear what to use when.

Reply:
Also what's "the console" here?